### PR TITLE
Induction CRUD Operation

### PR DESCRIPTION
**Pull request by BalajiNRaj**

### CODE REVIEW

File: src/main/java/com/example/utapCattle/service/repository/CattleRepository.java [hunk 1]
```diff
@@ -16,4 +18,12 @@ public interface CattleRepository extends JpaRepository<Cattle, Long> { // Use L
...
 	@Query(value = "select distinct eartag from cattle where isinductioncompleted is null or isinductioncompleted  = false;", nativeQuery = true)
 	List<String> getEarTagList();
+
+	/*
+	 * @Query(value = "SELECT * FROM cattle WHERE tagid = :tagId LIMIT 1",
+	 * nativeQuery = true) Cattle findByEartag(@Param("tagId") Integer tagId);
+	 */
+
+	@Query("SELECT c FROM Cattle c WHERE c.earTag = :earTag")
+	Optional<Cattle> findByEarTag(@Param("earTag") String earTag);
```
Comment (bharath2103) [inline src/main/java/com/example/utapCattle/service/repository/CattleRepository.java:28]: could you rename the methods as findEarTagsWithIncompleteInduction for clarity ?

File: src/main/java/com/example/utapCattle/service/repository/CattleRepository.java [hunk 2]
```diff
@@ -16,4 +18,12 @@ public interface CattleRepository extends JpaRepository<Cattle, Long> { // Use L
 
 	@Query(value = "select distinct eartag from cattle where isinductioncompleted is null or isinductioncompleted  = false;", nativeQuery = true)
```
Comment (bharath2103) [inline src/main/java/com/example/utapCattle/service/repository/CattleRepository.java:19]: also could you make this query as JPQL as you have did in findByEarTag() as below 

SELECT DISTINCT c.eartag FROM Cattle c WHERE c.isInductionCompleted IS NULL OR c.isInductionCompleted = FALSE

File: src/main/java/com/example/utapCattle/controller/InductionController.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+@CrossOrigin
+@RequestMapping("/induction")
+public class InductionController {
+
+	@Autowired
+	private InductionService inductionService;
+
+	@PostMapping("/save")
+	public ResponseEntity<InductionDto> saveCattle(@RequestBody final Induction induction) {
+		final InductionDto savedCattleDto = inductionService.saveInduction(induction);
```
Comment (bharath2103) [inline src/main/java/com/example/utapCattle/controller/InductionController.java:30]: please add logging messages

File: src/main/java/com/example/utapCattle/service/repository/CattleRepository.java (hunk 2 above)
Comment (BalajiNRaj) [reply src/main/java/com/example/utapCattle/service/repository/CattleRepository.java:19]: earTag and isinductioncompleted are in cattle table, 
we are retrieving the distinct earTags from cattle.
 Can we have it under CattleController?

**Pull request merged**